eZ/Publish/Core/Persistence/FieldTypeRegistry.php

```diff
@@ -91,7 +91,7 @@ public function register($identifier, $fieldType)
      *
      * @return \eZ\Publish\SPI\FieldType\FieldType
      */
-    protected function getCoreFieldType($identifier)
+    public function getCoreFieldType($identifier): FieldTypeInterface
```

Comment:
This change is required for https://github.com/ezsystems/ezpublish-kernel/pull/2632/files#diff-392b90a0512c4db62e37b27822a5bc20R72 but it's definitely not a clean solution. 

Instead of maybe we should introduce dedicated CoreFieldTypeRegistry or expose a method to check the type of FT?